sdk/servicebus/service-bus/src/connectionContext.ts

```diff
@@ -73,6 +73,10 @@ export interface ConnectionContext extends ConnectionContextBase {
    * Creates one if none exists in the cache
    */
   getManagementClient(entityPath: string): ManagementClient;
+  /**
+   * Whether the connection is closing.
+   */
+  isConnectionClosing(): boolean;
```

Comment:
Since this is getting to the public surface now, we should remove it from `ConnectionContextInternalMembers`

Reply:
(I'll push after the live tests finish running)